content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,545 @@
+description: User guide for ztunnel networking.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+## Introduction 
+
+This guide describes the functionality and usage of the Ztunnel proxy and Layer-4 networking functions using Istio Ambient mesh. We use a sample user journey to describe these functions. 
+
+The Ztunnel (Zero Trust Tunnel) component is a purpose-built per-node proxy for Istio ambient mesh. It is responsible for securely connecting and authenticating workloads within ambient mesh. Ztunnel is designed to focus on a small set of features for your workloads in ambient mesh such as mTLS, authentication, L4 authorization and telemetry, without terminating workload HTTP traffic or parsing workload HTTP headers. The ztunnel ensures traffic is efficiently and securely transported to the waypoint proxies, where the full suite of Istio’s functionality, such as HTTP telemetry and load balancing, is implemented.  Sometimes the term "Secure Overlay Networking" is also used informally to collectively describe the set of L4 networking functions implemented in an Ambient mesh via the Ztunnel proxy.
+
+It is expected that some production use cases of Istio in Ambient mode may be addressed solely via the L4 Secure overlay networking features whereas other use cases will additionally need advanced traffic management and L7 networking features for which the additional Waypoint proxies will need to be deployed. This is summarized in the following table. This guide focusses on functionality related to the baseline L4 and mTLS networking using ztunnel proxies. Other guides cover the advanced L7 networking functions that additionally require waypoint proxies. 
+
+
+| Application Deployment Use Case | Istio Ambient Mesh Configuration |
+| ------------- | ------------- |
+| Zero Trust networking via mutual-TLS, encrypted and tunneled data transport of all IP protocols, L4 authorization,  L4 telemetry | Baseline Ambient Mesh with ztunnel proxy networking |
+| Application requires L4 Mutual-TLS plus advanced Istio traffic management features (incl L4/ L7 VirtualService, L7 telemetry, L7 Authorization) | Full Istio Ambient Mesh configuration both ztunnel proxy and waypoint proxy based networking |
+
+
+[//]: # (There are additional subtle details on how functionality is split between the ztunnel and the waypoint proxies. These are described in later sections of this guide and other guides.)
+[//]: # ( As an example, if an Istio Authorization policy includes strictly L4 rules, then it is handled completely by ztunnel but if a policy includes only L7 rules or both L4 and L7 rules then it is handled by a Waypoint proxy. However when it comes to a VirtualService it is always handled by Waypoint proxies regardless of whether it has purely L4 rules, purely L7 rules or a mix of L4 and L7 rules. Other details on the functionality split between ztunnel and waypoint proxies will be described in later sections of this and other guides.) 
```

Comment:
not clear about if auth policy has both L4 and L7 if its only handled by waypoint proxy...  I think its still under debate what should happen based on this issue https://github.com/istio/istio/issues/47137.  Not sure if we want to include this section until its more clear.  I do agree its useful knowledge just maybe needs to wait.

Reply:
Authz is going through a lot of change in the community, might be a little too early to write.

Reply:
This text is commented out. We will have a small example of L4 authz similar to the getting started guide for now as discussed on the ambient slack channel. Additional enhancements will be documented once we approach close to beta status.